src/vscodePlatform.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
```

Comment:
Perhaps we should just add `alwaysStrict: true` to the tsconfig.json now that TypeScript 2.1 supports that flag?